neurobooth_os/iout/marker.py

```diff
@@ -4,13 +4,25 @@
 from pylsl import StreamInfo, StreamOutlet
 
 
-def marker_stream():
-    ''' Create marker stream to be pushed when needed with a string format:
-        "%s_%d_%timestamp" aka code string, number, time'''
+def marker_stream(name='Marker'):
+    """Create marker stream to be pushed when needed with a string format:
+        "%s_%d_%timestamp" aka code string, number, time
```

Comment:
multiline comments are not allowed. First one line, then a space line then the rest